lib/Analysis/Allocation.cpp

```diff
@@ -70,6 +60,8 @@ getScratchConfigForCvtLayout(triton::gpu::ConvertLayoutOp op, unsigned &inVec,
         std::max(std::min<unsigned>(srcTy.getShape()[d], srcShapePerCTA[d]),
                  std::min<unsigned>(dstTy.getShape()[d], dstShapePerCTA[d]));
   }
+  if (rank == 1)
+    return paddedRepShape;
```

Comment:
If `rank == 1`, we can skip `paddedRepShape[paddedDim] += pad;` (Line 69) ?

Reply:
Yep, the padding is only there to avoid shared memory bank conflicts, so it should be ok to return `paddedRepShape` after it gets actually padded